includes/class-wc-privacy.php

```diff
@@ -370,7 +370,42 @@ protected static function delete_inactive_accounts_query( $timestamp, $limit = 2
 			}
 
 			foreach ( $user_ids as $user_id ) {
-				wp_delete_user( $user_id );
+				$posts_made_prior = get_posts(
+					array(
+						'post_type' => array( 'post', 'page' ),
```

Comment:
This is not enough, also we may delete some 3rd party content as well, so probably better to include a filter.

You can include this before the foreach:

```php
$builtin_post_types = get_post_types( array( '_builtin' => true ) );
unset( $builtin_post_types['user_request'] ); // Exclude user_request from data migration.

/**
 * Filter post types that should be assign to an admin user prior user exclusion.
 *
 * @param array $post_types List of post types to assign to an admin. Defaults to WordPress builtin post types.
 */
$post_types_to_assign = apply_filters( 'woocommerce_delete_inactive_account_post_types_to_assign', array_values( $builtin_post_types ) );
```

After you just need to update the query to:
```php
'post_type' => $post_types_to_assign,
```